.github/CODEOWNERS

```diff
@@ -4,3 +4,8 @@
 # review when someone opens a pull request.
 
 *       @department-of-veterans-affairs/frontend-review-group
+
+# VSA
+src/applications/proxy-rewrite/redirects/crossDomainRedirects.json @department-of-veterans-affairs/vsa-frontend
+src/applications/find-va-forms @department-of-veterans-affairs/vsa-frontend
+src/platform/site-wide @department-of-veterans-affairs/vsa-frontend
```

Comment:
I think this represents the bulk of what your team owns @ncksllvn . it includes the feature toggle stuff but I think that's fine for now. We aren't actively developing that and we can move that component into another platform folder.

Reply:
@rianfowler It would be awesome if we could get `src/applications/yellow-ribbon` too if that's possible! 🙂

Reply:
@ncksllvn do you think there will be another app we maintain for the search epic?

Reply:
We can add whatever apps you want as long as you have sole ownership of it.